packages/plugin-ext/src/main/electron-browser/webview/electron-webview-frontend-module.ts

```diff
@@ -0,0 +1,23 @@
+/********************************************************************************
+ * Copyright (C) 2020 Ericsson and others.
```

Comment:
Don’t we already have an electron entry point for the plugin extension?

Reply:
We don't have an Electron-frontend entry point for the plugin extension.